solutions/bronze/usaco-1276.mdx

```diff
@@ -131,9 +131,10 @@ min_cost = float(
 	"inf"
 )  # the minimum amount of money needed to keep all cows comfortable
 
-
-# Based on 'uses', determine if the current subset of air conditioners suffices
-# the constraints, and if so, update the minimum cost
+"""
+Based on 'uses', determine if the current subset of air conditioners suffices
+the constraints, and if so, update the minimum cost
+"""
```

Comment:
yk docstrings go below the function declaration right

Reply:
I had no idea about it